repository-data/webfiles/src/main/resources/site/src/scss/components/_detail-list-grid.scss

```diff
@@ -66,6 +66,14 @@
         }
     }
 
+    &--regular {
+        .detail-list {
+            border-bottom: 1px solid $col-14;
+            margin-top: $gutter * 0.5 + 2px;
+            padding-bottom: $gutter * 0.5;
+        }
+    }
+
```

Comment:
Should this be added/from the UI toolkit?

Reply:
I am afraid our knowledge of the UI is very limited and tackling this may delay the release. Is it ok if we do not fix this now?